content/community/coc/index.md

```diff
@@ -5,121 +5,99 @@ title: "Galaxy Project Code of Conduct"
+  - Unwelcome comments or jokes, to an audience or personally, regarding another person’s lifestyle choices and practices, including those related to food, health, religion, parenting, drugs, and employment.
+  - Mocking or insulting another person’s intellect, work, perspective, or question/comment.
+  - Deliberate misgendering or use of ‘dead’ or rejected names.
+  - Gratuitous or off-topic sexual images or behavior in spaces where they’re not appropriate.
+  - Physical contact and simulated physical contact (e.g., textual descriptions like “*hug*” or “*backrub*”) without consent or after a request to stop.
+  - Threats of violence.
+  - Incitement of violence towards any individual, including encouraging a person to commit suicide or to engage in self-harm.
+  - Deliberate intimidation.
+  - Stalking or following.
+  - Harassing photography or recording, including logging online activity for harassment purposes.
+  - Sustained disruption of discussion.
+  - Unwelcome sexual attention.
+  - Patterns of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others.
+  - Continued one-on-one communication after requests to cease.
+  - Deliberate “outing” of any aspect of a person’s identity without their consent, except as necessary to protect vulnerable people from intentional abuse.
+  - Malicious publication of otherwise private communication, or private personal information (“doxing” or “outing”).
+
+
+The Galaxy Community prioritizes marginalized people’s safety over privileged people’s comfort. The [Galaxy CoC Committee](#the-galaxy-coc-committee) reserves the right not to act on complaints regarding:
+- ‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’.
+- Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you.”
+- Communicating in a ‘tone’ you don’t find congenial.
+- Criticizing racist, sexist, cissexist, or otherwise oppressive behavior or assumptions.
+
+
+### Reporting
+
+If you are being harassed by a member of the Galaxy Community, notice that someone else is being harassed, or have any other concerns, please send a report via [this form](https://forms.gle/gEMubTrW9ifwemqe6) that will be forwarded to the [Galaxy CoC Committee](#the-galaxy-coc-committee). Alternatively, you can [email the Code of Conduct Committee](mailto:code-of-conduct@lists.galaxyproject.org). If you are uncomfortable reporting to the Code of Conduct committee as a group, you can contact any individual committee member via email. If the person who is harassing you is on the team, they will recuse themselves from handling your incident. We will respond as promptly as we can.
+
+To protect volunteers from abuse and burnout, we reserve the right to reject any report we believe to have been made in bad faith. 
```

Comment:
Do we need to specify what good/bad faith consititutes ?

Reply:
That's a good point. And a hard question, I think it's hard to define. We would need to discuss this in the first committee meeting.